### PR TITLE
*: Refactor the structure

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

### CODE REVIEW

**Comment by gaocegege** [comment]
```
.
├── cmd
│   ├── cli
│   ├── earlystopping
│   │   └── medianstopping
│   ├── manager
│   └── suggestion
│       ├── bayesianoptimization
│       ├── grid
│       ├── hyperband
│       └── random
├── dlk
│   ├── dlkctl
│   │   └── utils
│   └── dlkmanager
│       ├── api
│       ├── configs
│       └── datastore
├── docs
├── examples
├── manifests
├── modeldb
├── pkg
│   ├── api
│   │   └── python
│   ├── db
│   │   └── test
│   ├── earlystopping
│   ├── manager
│   │   ├── modelstore
│   │   │   └── modeldb
│   │   │       └── model_d_b_service-remote
│   │   ├── visualise
│   │   │   └── tensorboard
│   │   │       └── manifest_template
│   │   └── worker_interface
│   │       ├── dlk
│   │       ├── kubernetes
│   │       └── nvdocker
│   ├── mock
│   │   ├── api
│   │   ├── db
│   │   ├── modelstore
│   │   └── worker
│   └── suggestion
│       ├── bayesianoptimization
│       │   └── src
│       │       ├── acquisition_func
│       │       ├── global_optimizer
│       │       ├── model
├── scripts
└── test

```

Does it work for you? @ddysher ?

I did not refactor dlk since we have an issue to migrate it into the worker_interface #66

**Comment by gaocegege** [comment]
/assign @YujiOshima @ddysher 

I prefer to merge it after CI works. But it is time to review now.

File: scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+set -o nounset
+set -o pipefail
+
+PREFIX="katib"
+CMD_PREFIX="cmd"
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+
+cd ${SCRIPT_ROOT}
+# docker build -t ${PREFIX}/vizier-core -f ${CMD_PREFIX}/manager/Dockerfile .
```
Comment (YujiOshima) [inline scripts/build.sh]: Why you commented out suggestions?
And please add Bayesian optimization suggestion build.

**Comment by YujiOshima** [comment]
https://github.com/kubeflow/katib/pull/65/files#diff-fb322a36346f22df69b97c6f2b9d5930L24
nit `./deploy.sh` -> `./scripts/deploy.sh`

**Comment by YujiOshima** [comment]
Please remove katib-manager binary file.

**Comment by gaocegege** [comment]
Addressed, PTAL

**Comment by ddysher** [comment]
yeah, it works.  thanks!
> Does it work for you? @ddysher ?

File: scripts/build.sh (hunk 1 above)
Comment (gaocegege) [reply scripts/build.sh]: This is for test, sorry for it. OK

**Comment by YujiOshima** [comment]
@gaocegege Thanks!
/lgtm

**Pull request merged**